### PR TITLE
docs: Clarify that randomElements() does not repeat input elements

### PR DESCRIPTION
**Pull request by sustmi**
It was not clear for me whether randomElements() repeats input elements or not until I looked into the implementation.

### CODE REVIEW

File: src/Faker/Provider/Base.php [hunk 1]
```diff
@@ -156,7 +156,7 @@ public static function randomAscii()
     }
 
     /**
-     * Returns random elements from a provided array
+     * Returns random elements from a provided array (not repeating input indices)
```
Comment (fzaninotto) [inline src/Faker/Provider/Base.php]: I don't understand your comment

Comment (sustmi) [inline src/Faker/Provider/Base.php]: What about this?
> Returns randomly ordered subsequence of $count elements from a provided array

I want for the other users to be clear that the function does not repeat input elements. The current comment does not say it.

Comment (fzaninotto) [inline src/Faker/Provider/Base.php]: ok

**Comment by sustmi** [comment]
I updated the pull request with the new sentence.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**